addons/account/migrations/12.0.1.1/pre-migration.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2018 Eficent <http://www.eficent.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+_column_copies = {
+    'account_journal': [
+        ('bank_statements_source', None, None),
+    ],
+    'account_payment_term_line': [
+        ('option', None, None),
+    ],
+}
+
+_column_renames = {
+    'account_tax': [
+        ('tax_adjustment', None),
+    ],
+    'account_tax_template': [
+        ('tax_adjustment', None),
+        ('cash_basis_account', 'cash_basis_account_id'),
```

Comment:
No need for this, but call `rename_fields` instead. And do it for `account.tax`, not for `account.tax.template`, which is a new field.

Reply:
The rename is done for both `account.tax` and `account.tax.template`.